lib/beaker/tasks/quick_start.rb

```diff
@@ -1,49 +1,93 @@
 require 'beaker-hostgenerator'
 
+VAGRANT  = ['ubuntu1404-64default.mdcal-ubuntu1404-64af', '--hypervisor=vagrant',
+            '--global-config=box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm']
```

Comment:
Does this work? 

This is going to need to follow the custom `{key=value,...}` syntax, so I think we need to wrap it in braces like so:
```
--global-config={box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm}
```